app/javascript/packs/Onboarding.jsx

```diff
@@ -0,0 +1,43 @@
+import { h, render } from 'preact';
+import { getUserDataAndCsrfToken } from '../chat/util';
+import getUnopenedChannels from '../src/utils/getUnopenedChannels';
+
+HTMLDocument.prototype.ready = new Promise(resolve => {
+  if (document.readyState !== 'loading') {
+    return resolve();
+  }
+  document.addEventListener('DOMContentLoaded', () => resolve());
+  return null;
+});
+
+function isUserSignedIn() {
+  return (
+    document.head.querySelector(
+      'meta[name="user-signed-in"][content="true"]',
+    ) !== null
+  );
+}
+
+function renderPage() {
+  import('../src/Onboarding')
+    .then(({ default: Onboarding }) =>
+      render(<Onboarding />, document.getElementById('top-bar')),
+    )
+    .catch(error => {
+      // eslint-disable-next-line no-console
+      console.error('Unable to load onboarding', error);
```

Comment:
I see that there is airbrake for the back-end, but from what I can tell, we don't use any client-side error reporting services. Consider a package for handling client-side errors as we'll creating more and more in the front-end now.